src/index.js

```diff
@@ -30,11 +30,27 @@ if (!isProduction) {
 }
 
 function run() {
-  const promises = meetups.map(meetup => getMeetupEventsData(`https://api.meetup.com/${meetup}/events`));
   const resolveAllPromises = isProduction ? resolveMeetupEventsDataS3 : resolveMeetupEventsDataLocal;
+  const eventsData = getMeeptupEventsData();
+  const groupsData = getMeeptupGroupsData();
+  let groupImages = {};
 
-  Promise.all(promises)
-    .then(resolveAllPromises)
+  Promise.all([eventsData, groupsData])
+    .then(function (results) {
+      results[0] = JSON.parse(formatResults(results[0]));
+
+      // Build object containing group name, and associated group image.
+      results[1].map(group => {
+        groupImages[group.urlname] = group.group_photo !== undefined ? group.group_photo.highres_link : null;
+      });
+
+      // Append this image to event
+      results[0].map(event => {
+        event.group.group_photo = groupImages[event.group.urlname]; // eslint-disable-line
```

Comment:
@Peter-holyoke 
What linting error is being suppressed here?  Generally I would prefer to state the explicit rule as opposed to just a blanket off switch for all linting